docs/concepts.md

```diff
@@ -64,7 +77,41 @@ A frame is not something that the user interacts with and is used to derive furt
 
 (in `node_modules/@replayio/protocol/js/protocol/Pause.d.ts`)
 
-TODO why "Frame" exists and how it's used, what sort of info is derived using it
+
+#### why `Frame` exists and how it's used, what sort of info is derived using it
+
+`Frame` is a key concept used in the Replay DevTools codebase to describe the stack frames of the JavaScript program. A `Frame` object is returned as part of a `Pause` object when the user requests the state of the JavaScript program at a specific ExecutionPoint.
+
+A `Frame` contains information about the current state of a function call stack frame. It includes details such as the function name, the location of the function in the code, and the values of its arguments and local variables at the specific `ExecutionPoint`.
+
+The `Frame` concept is important because it helps the user understand the flow of the program and the sequence of function calls that led to the current state. The DevTools can display the Frame information in a call stack, allowing the user to see the sequence of function calls that led to the current state of the program.
+
+<details>
+  <summary>There are four types of `Frames` in the Replay DevTools codebase:</summary>
+  <ul>
+    <li><strong>Call:</strong> represents a function call.</li>
+    <li><strong>Function location: </strong> represents the global scope.</li>
+    <li><strong>Arguments: </strong> represents a module scope.</li>
+    <li><strong>`this` object:</strong> represents an eval function call.</li>
+  </ul>
```

Comment:
I don't understand the way this list is worded. (For instance, why is "arguments" shown as the label for module scope?)

In the code, these are the four frame types:
```ts
export declare type FrameType = "call" | "global" | "module" | "eval";
```